lib/Twig/Template.php

```diff
@@ -321,33 +321,6 @@ public function getBlocks()
     }
 
     /**
-     * Returns the template source code.
-     *
-     * @return string|null The template source code or null if it is not available
-     */
-    public function getSource()
```

Comment:
``getSource`` should still be defined in this class (to be overwritten by child classes), to make it part of the ``Twig_Template`` API (and to make it part of the blacklisted method names when generating methods dynamically in compiled templates).

In Twig 1.x, we should make it concrete (returning an empty string). In Twig 2.x, it can become an abstract method